library/Ivoz/Provider/Domain/Model/Voicemail/VoicemailDto.php

```diff
@@ -4,6 +4,8 @@
 
 class VoicemailDto extends VoicemailDtoAbstract
 {
+    public const CONTEXT_MY_VOICEMAILS = "myVoicemails";
```

Comment:
Custom context are only needed when we've many endpoints of the same type (get one, get many, post, put, etc. )

Reply:
In deed introduced more than one endpoint. Kai requested me to hide the user field on all responses. Is there a better way to achieve that?